src/style.css

```diff
@@ -34,9 +34,9 @@ li {
   border-bottom: solid 1px rgba(194, 194, 194, 0.4);
 }
 
-.list-items-text {
-  opacity: 70%;
-}
+/* .list-items-text {
+  opacity: 75%;
+} */
```

Comment:
- [ ] It will be nice to remove all commented code from your code-base

Reply:
Well done

Reply:
Well done

Reply:
Well done